src/features/import/components/ImportDialog/elements/ImportMessageList/index.tsx

```diff
@@ -31,6 +37,24 @@ const ImportMessageList: FC<Props> = ({
     onAllChecked(numChecked == warningCount);
   }, [numChecked]);
 
+  const formatProblemIndex = problems.findIndex(
+    (problem) => problem.kind == ImportProblemKind.INVALID_FORMAT
+  );
+  if (formatProblemIndex != -1) {
+    const problem = problems[formatProblemIndex] as ImportFieldProblem;
+    if (
+      problem.field == 'phone' &&
+      getCountries().findIndex(
+        (iso) => store.getState().organizations.orgData.data?.country == iso
+      ) == -1
```

Comment:
Could this be saved in a named constant, to make it a bit easier to read?

Reply:
I have some thoughts but I guess it will be simplified after moving to `predictProblems` which already has the country code available :)